### PR TITLE
Matching Page

### PR DESCRIPTION
**Pull request by wclui**

### CODE REVIEW

File: server/matches.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from mysql.connector import errorcode
+from connect import connectToDB
+
+def matchUser(userId):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            sql = "SELECT firstname from Profile where userId in (SELECT distinct shownUser from PotentialMatch where currentUser = %s and matchDecision = True)"
+            cursor.execute(sql, (int(userId),))
```
Comment (mtlssica) [inline server/matches.py:29]: Do you have to make it (int(userId)) or if u can just put in userId instead?

File: server/matches.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import mysql.connector
+from mysql.connector import errorcode
+from connect import connectToDB
+
+def matchUser(userId):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            sql = "SELECT firstname from Profile where userId in (SELECT distinct shownUser from PotentialMatch where currentUser = %s and matchDecision = True)"
```
Comment (mtlssica) [inline server/matches.py]: is select distinct required? The table should be in a way that there's no duplicated shownUser

File: src/Matches.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            })
+    }
+
+    render(){
+        let matchingSection;
+        if (this.state.matchesExist == "exists"){
+            let userItems = [];
+            for (var i = 0; i < this.state.potentialUserId.length; i++){
+                var pos_user = this.state.potentialUserId[i];
+                userItems.push(
```
Comment (mtlssica) [inline src/Matches.js:45]: Also, are you just planning to show the user ID, or maybe in later sprints also get at least the name of the user or their description?

File: server/matches.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def insertConvo(userOne, userTwo):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            sql = "IF EXISTS (SELECT * from PotentialMatch p1 where \
+                (currentUser = %s and matchDecision = 1 and potentialUser = %s) AND \
+                    (SELECT * from PotentialMatch p2 where currentUser = %s and matchDecision = 1 and potentialUser = %s) \
```
Comment (echanlee) [inline server/matches.py:13]: Capitalize key words like where

File: src/Matches.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                Sorry, No one wants to be your friend for now. Please try again later :(
+            </h2>
+        }
+        else {
+            matchingSection = <h2></h2>
+        }
+        return(
+            <div id='Matches-section'>
+                <h2 id='Matches-header'>Matches</h2>
+                <div>{this.state.userId}</div>
```
Comment (echanlee) [inline src/Matches.js:70]: Why are you outputting their id

**Pull request merged**